images/kamu-web-ui/Dockerfile

```diff
@@ -15,3 +18,5 @@ RUN cd /usr/share/nginx/html && \
 
 COPY runtime-config.json /usr/share/nginx/html/assets/runtime-config.json
 COPY runtime-config-single-tenant.json /usr/share/nginx/html/assets/runtime-config-single-tenant.json
+
+USER nginx
```

Comment:
Hm, how does this work?
- We are downloading and extracting the application files under `0 / root` user
- We don't do `chown`
- But somehow `nginx` user can read files belonging to root in the end?

Reply:
Static data doesn't have to belong to `nginx` to be served, it is sufficient for the `nginx` user to only have read permissions to be able to serve the static files. Actually it even shouldn't be writable by the `nginx` user from a security standpoint.

It works in the same way in existing container image, where the static data is owned by the `root` user. 
When the `nginx` master process is started, it runs as `root`, but when the master process spawns worker processes, the EUID of worker processes is lowered to the `nginx` user, which is controlled by the `user nginx;` directive in the `/etc/nginx/nginx.conf` file.